dev/Telemetry/RuntimeProfiler.cpp

```diff
@@ -111,62 +174,58 @@ namespace RuntimeProfiler {
                     //  up on the next FireEvent().
                     continue;
                 }
-            
-                //  Zeroing out AND getting value.
-                cHits = InterlockedExchange(m_Counts[ii].pInstanceCount, 0);
-            
-                if (0 != cHits)
+
+                //  Zeroing out AND getting current value.
+                cHits = (UINT)(::InterlockedExchange(m_Counts[ii].pInstanceCount, 0));
```

Comment:
[static_cast](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es49-if-you-must-use-a-cast-use-a-named-cast)